onnxruntime/test/framework/execution_frame_test.cc

```diff
@@ -496,14 +496,14 @@ TEST(ExecutionFrameTestInit, InitializerAsOutput) {
 
 #if !defined(DISABLE_SPARSE_TENSORS)
 TEST(ExecutionFrameTestInit, SparseInitializerAsOutput) {
-  const std::vector<int64_t> dense_shape{3, 3};
-  std::vector<float> dense_data = {
-      0, 0, 1.764052391052246f,
-      0.40015721321105957f, 0, 0.978738009929657f,
-      0, 0, 0};
+  constexpr std::array<int64_t, 2> dense_shape{3, 3};
+  // constexpr std::array<float, 9> dense_data = {
```

Comment:
do we need to keep the commented out stuff?

Reply:
I found it useful as it provides context on what the dense data being returned as sparse looks like. however a comment explaining that would be very helpful